config/crd/external/authorino.kuadrant.io_authconfigs.yaml

```diff
@@ -53,14 +53,19 @@ spec:
         description: AuthConfig is the schema for Authorino's AuthConfig API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Is it the just crd update?
which authorino version is it?

Reply:
Yes, it is an update to match the version of [the imported library](https://github.com/opendatahub-io/odh-model-controller/blob/incubating/go.mod#L12).

I took it from here: https://github.com/Kuadrant/authorino/blob/main/install/crd/authorino.kuadrant.io_authconfigs.yaml

Reply:
okay, thanks.